actors/builtin/storage_power/storage_power_actor.go

```diff
@@ -294,32 +295,30 @@ func (a *StoragePowerActor) OnMinerEnrollCronEvent(rt Runtime, eventEpoch abi.Ch
 	return &adt.EmptyValue{}
 }
 
-func (a *StoragePowerActor) ReportConsensusFault(rt Runtime, blockHeader1, blockHeader2 []byte, slashee addr.Address, faultEpoch abi.ChainEpoch, faultType ConsensusFaultType) *adt.EmptyValue {
+func (a *StoragePowerActor) ReportConsensusFault(rt Runtime, blockHeader1, blockHeader2 []byte, target addr.Address, faultEpoch abi.ChainEpoch, faultType ConsensusFaultType) *adt.EmptyValue {
 	// TODO: determine how to reward multiple reporters of the same fault within a single epoch.
 
 	isValidConsensusFault := rt.Syscalls().VerifyConsensusFault(blockHeader1, blockHeader2)
 	if !isValidConsensusFault {
 		rt.Abort(exitcode.ErrIllegalArgument, "spa.ReportConsensusFault: unverified consensus fault")
 	}
 
-	slasherAddr := rt.ImmediateCaller()
-	var amountToSlasher abi.TokenAmount
+	reporter := rt.ImmediateCaller()
 	var st StoragePowerActorState
-	rt.State().Transaction(&st, func() interface{} {
-		claimedPower, powerOk, err := getStoragePower(adt.AsStore(rt), st.ClaimedPower, slashee)
+	reward := rt.State().Transaction(&st, func() interface{} {
+		store := adt.AsStore(rt)
+		claimedPower, powerOk, err := getStoragePower(store, st.ClaimedPower, target)
 		if err != nil {
 			rt.Abort(exitcode.ErrIllegalState, "spa.ReportConsensusFault failed to read claimed power for fault: %v", err)
 		}
 		if !powerOk {
 			rt.Abort(exitcode.ErrIllegalArgument, "spa.ReportConsensusFault: miner already slashed")
 		}
-		Assert(claimedPower.GreaterThan(big.Zero()))
+		Assert(claimedPower.GreaterThanEqual(big.Zero()))
```

Comment:
@zixuanzh I think this was a bug, a miner might have dropped to zero power subsequent to misbehaving.

Reply:
possible 👍